### PR TITLE
Update Beta documentation index#284

### PR DESCRIPTION
**Pull request by alansunny3**
## Summary
Revise and correct documentation to remove alpha warnings and update references with beta.

## Details
All references to Alpha phase has been replaced with Beta in order to align with Beta phase announcement. 

ticket:
https://app.zenhub.com/workspaces/obol-workspace-648c1170ee94656741353327/issues/gh/obolnetwork/obol-docs/284

### CODE REVIEW

**Review by thomasheremans — APPROVED** [review]
Left some comment for the monitoring page, otherwise LGTM

File: docs/int/quickstart/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Quickstart Guides
 
 :::caution
-Charon is in an alpha state and should be used with caution according to its [Terms of Use](https://obol.tech/terms.pdf).
+Charon is in an beta state and should be used with caution according to its [Terms of Use](https://obol.tech/terms.pdf).
```
Comment (OisinKyne) [inline docs/int/quickstart/index.md:4]:
```suggestion
Charon is in a beta state and should be used with caution according to its [Terms of Use](https://obol.tech/terms.pdf).
```

File: docs/int/quickstart/quickstart-exit.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ import TabItem from '@theme/TabItem';
 # Exit a DV
 
 :::caution
-Charon is in an alpha state and should be used with caution according to its [Terms of Use](https://obol.tech/terms.pdf).
+Charon is in an beta state and should be used with caution according to its [Terms of Use](https://obol.tech/terms.pdf).
```
Comment (OisinKyne) [inline docs/int/quickstart/quickstart-exit.md:12]:
```suggestion
Charon is in a beta state and should be used with caution according to its [Terms of Use](https://obol.tech/terms.pdf).
```

File: docs/int/quickstart/quickstart-mainnet.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ description: Run a cluster on mainnet
 # Run a DV on mainnet
 
 :::caution
-Charon is in an alpha state, and you should proceed only if you accept the risk, the [terms of use](https://obol.tech/terms.pdf), and have tested running a Distributed Validator on a testnet first.
+Charon is in an beta state, and you should proceed only if you accept the risk, the [terms of use](https://obol.tech/terms.pdf), and have tested running a Distributed Validator on a testnet first.
```
Comment (OisinKyne) [inline docs/int/quickstart/quickstart-mainnet.md:9]:
```suggestion
Charon is in a beta state, and you should proceed only if you accept the risk, the [terms of use](https://obol.tech/terms.pdf), and have tested running a Distributed Validator on a testnet first.
```

**Comment by OisinKyne** [comment]
Suggesting we close this PR as most of the changes went in on #288, hence the conflicts. Monitoring guide could still do with being updated to refer to a cluster with a generic name rather than Alpha M1 though.

**Pull request closed without merging**